### ISSUE TITLE
onPanResponderMove: GestureState.moveX/moveY incorrect on first movement

### ISSUE THREAD

**Issue by raineorshine** [necolas/react-native-web#1853]
**The problem**

The `GestureState` passed to the `onPanResponderMove` handler incorrectly returns `moveX: -1, moveY: -1` when it is first called.

```js
this.panResponder = PanResponder.create({
  onMoveShouldSetPanResponder: () => true,
  onPanResponderMove: (e, gestureState) => {
    console.log(`moveX,moveY: ${gestureState.moveX},${gestureState.moveY}`);
  }
});
```

**How to reproduce**

Simplified test case: https://codesandbox.io/s/mutable-night-tm2uq?file=/src/App.js

Steps to reproduce:
1. Click and drag (pan) in the pink box
2. Open the Console and check the first line of output

**Expected behavior**

The `moveX,moveY` coordinates should correspond to where the movement started. Instead, the first trigger always returns `-1,-1`.

**Environment (include versions). Did this work in previous versions?**

This worked in `v0.14.5`. Untested between `v0.14.5` and `v0.14.10`.

* React Native for Web (version): `0.14.10`
* React (version): `16.14.0`
* Browser: Chrome and Safari

**Comment by necolas** [comment]
This regression was likely introduced by https://github.com/necolas/react-native-web/commit/d2e6c29e259856b81d0466217754c1107a44f014

**Comment by necolas** [comment]
Thanks for being patient
This should be fixed in 0.15.6
https://codesandbox.io/s/flamboyant-fermi-pm63o
